services-geojson/src/main/java/com/mapbox/geojson/Feature.java

```diff
@@ -258,8 +258,11 @@ public Geometry geometry() {
      * @return a {@link JsonObject} which holds this features current properties
      * @since 1.0.0
      */
-    @Nullable
+    @NonNull
```

Comment:
Looks like a good tool. We could do that.

Reply:
Well, on second though the filed "properties" can be null. It is the result of the properties() call that cannot be null when we are calling it. That is why it's return result should be @NonNull and it should through an exception if we try to access it.

This is the same behavior as it was before, except that before it would crash on access and not through explicit exception. I think it is cleaner to through an exception if null and after that assume that it is not null and not worry about multiple null checked (or places to crash as before)

Reply:
Okay, makes sense now.